pkgs/development/compilers/openspin/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchurl, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  name = "openspin-${version}";
+  version = "2018-02-26";
+
+  src = fetchFromGitHub {
+    owner = "parallaxinc";
+    repo = "OpenSpin";
+    rev = "7bbc030c2967e5d266a076c0a1a5ce4c21ae7b06";
+    sha256 = "19xxfapvyskimpkg9g76rd9j4gcfikmpdx2mf599w1yzwcnx10s9";
+  };
+
+  doCheck = true;
+
+  postPatch = ''
+    sed -i -e '1,18d' -e '20d' -e 's/  BUILD=/BUILD=/' Makefile
```

Comment:
This will not be very stable. You can just use `makeFlags = [ "CROSS=none" ];` if something in Nixpkgs sets the variable.

Reply:
I attempted this but it wasn't successful because the code itself re-defines, ignoring the set environment.

I'll make a request upstream, but bearing in mind how infrequently changes are made upstream it's not something I would want to hold for.

I'll commit to updating the script if I can convince them of a better way.

Reply:
@jtojnar  - Upstream accepted my PR.
I've updated to their latest version on github which was after my PR was applied.

Resolved?